### ISSUE TITLE
`tom-select` v2.4 import breaks actiontext

### ISSUE THREAD

**Issue by TrevorHinesley** [excid3/madmin#232]
Starting at [v2.4.0](https://github.com/orchidjs/tom-select/releases/tag/v2.4.0) that was released last week (new maintainer has taken it on apparently), the import breaks with this console error:

```
Uncaught TypeError: Failed to resolve module specifier "@orchidjs/sifter". Relative references must start with either "/", "./", or "../".
```

Resolved by changing the importmap in the `_javascript.html.erb` partial from:

```
https://unpkg.com/tom-select/dist/esm/tom-select.complete.js
```

to:

```
https://unpkg.com/tom-select@2.3.0/dist/esm/tom-select.complete.js
```

But unsure if that's how you'd like to resolve it, so I thought I'd drop this here for posterity's sake.

**Comment by excid3** [comment]
Thanks! This might be fixed by #230

My long-term plan is to remove tom-select entirely and use Hotwire Combobox instead in the long run.

**Comment by TrevorHinesley** [comment]
Yep looks like it will be. Awesome, combobox looks interesting!
